cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -11,11 +11,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	dto "github.com/prometheus/client_model/go"
-	"k8s.io/klog/v2"
 )
 
 var COPY_DISK_RE = regexp.MustCompile(`^.*Copying disk (\d+)/(\d+)`)
-var DISK_PROGRESS_RE = regexp.MustCompile(`^..\s*(\d+)% \[.*\]`)
+var DISK_PROGRESS_RE = regexp.MustCompile(`.+ (\d+)% \[[*-]+\]`)
```

Comment:
are we sure we need to change the regex? i looked on this regex and the progress given in the log and it looked correct.

Reply:
@liranr23 this change was inspired from https://github.com/oVirt/vdsm/pull/402
the problem with the current regex is that it catches lines that are unrelated to the progress